jvm/kotlin-http/src/main/kotlin/dev/restate/sdk/examples/Greeter.kt

```diff
@@ -0,0 +1,37 @@
+package dev.restate.sdk.examples
+
+import dev.restate.sdk.core.CoreSerdes
+import dev.restate.sdk.core.StateKey
+import dev.restate.sdk.examples.generated.*
+import dev.restate.sdk.examples.generated.GreeterProto.GreetRequest
+import dev.restate.sdk.examples.generated.GreeterProto.GreetResponse
+import dev.restate.sdk.kotlin.RestateCoroutineService
+import dev.restate.sdk.http.vertx.RestateHttpEndpointBuilder
+import kotlinx.coroutines.Dispatchers
+
+class Greeter :
+    // Use Dispatchers.Unconfined as the Executor/thread pool is managed by the SDK itself.
+    GreeterGrpcKt.GreeterCoroutineImplBase(Dispatchers.Unconfined),
+    RestateCoroutineService {
+
+  companion object {
+    private val COUNT = StateKey.of("count", CoreSerdes.INT)
+  }
```

Comment:
Nice! I was going to comment that a companion object is a more Kotlin-idiomatic way of declaring constants :-)